routes/web.php

```diff
@@ -2,21 +2,12 @@
 
 use Illuminate\Support\Facades\Route;
 
-/*
-|--------------------------------------------------------------------------
-| Web Routes
-|--------------------------------------------------------------------------
-|
-| Here is where you can register web routes for your application. These
-| routes are loaded by the RouteServiceProvider within a group which
-| contains the "web" middleware group. Now create something great!
-|
-*/
-
 Route::get('/', function () {
     return view('welcome');
 });
 
 Auth::routes();
 
 Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
```

Comment:
I suggest ipangan ni og Dashboard nalang, ato lang ibase ang naming sa specification para dali masabtan. and Sa files related ani kay Dashboard nato iname.

Controller => DashboardController.php
View =>  dashboard.blade.php


and then under dashboard view we will create partial files.
   => activities.blade.php
   => user-details.blade.php



Purpose of this kay mas organize sya tanawn, mas dali i debug, mas limpyo. We can discuss this tomorrow

Reply:
Done